scripts/run.sh

```
@@ -3,9 +3,23 @@
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 VOX_ENV="${SCRIPT_DIR}/../.env.local"
+ROOT_DIR="${SCRIPT_DIR}/.."
 
 if [ -f "$VOX_ENV" ]; then
-    export $(grep -E '^(ELEVENLABS_API_KEY|GEMINI_API_KEY|OPENROUTER_API_KEY|DEEPGRAM_API_KEY|OPENAI_API_KEY)=' "$VOX_ENV" | xargs)
+    while IFS= read -r line; do
+        [ -n "$line" ] && export "$line"
```

Shellcheck (0.11.0)

[warning] This does not export 'line'. Remove $/${} for that, or use ${var?} to quiet.

(SC2163)